proto/mixer.proto

```diff
@@ -625,6 +623,20 @@ message GetStatSeriesResponse {
   map<string, double> series = 1;
 }
 
+// Request for GetStatAll service.
+message GetStatAllRequest {
+  // dcids of the place.
+  repeated string places = 1;
+  // dcids of the stat var.
+  repeated string stat_vars = 2;
+}
+
```

Comment:
comments for the response proto? especially to note what the string keys are.

Reply:
done